faasmcli/faasmcli/tasks/dev.py

```diff
@@ -1,12 +1,19 @@
 from os import makedirs
 from os.path import exists
-from shutil import rmtree
 from subprocess import run
 
 from invoke import task
 
 from faasmcli.util.env import PROJ_ROOT, FAASM_BUILD_DIR
 
+DEV_TARGETS = [
```

Comment:
Unrelated improvement for a shortcut for compiling all the development-related targets